test/aqua/operators/test_state_op_meas_evals.py

```diff
@@ -79,6 +80,12 @@ def test_coefficients_correctly_propagated(self):
             sampler = CircuitSampler(q_instance).convert(~StateFn(op) @ state)
             self.assertAlmostEqual(sampler.eval(), 1+0j)
 
+        with self.subTest('ListOp evaluated to List'):
+            state = ListOp([Plus, Zero])
+            sampler = CircuitSampler(q_instance).convert(~state @ state)
+            self.assertListEqual(sampler.eval(),
+                                 [[1.0, 0.6987712429686843], [0.7064159097160821, 1.0]])
```

Comment:
Comparing to plain numbers is never a good idea 😉 Could you change this to just test if the first operand is still a measurement after going through the circuit sampler? Otherwise LGTM.

Reply:
But all other tests in the module compare ```.eval()``` result to plain numbers.

Reply:
@Cryoris I changed the unit test in the way you suggested. Thanks for review.